Demo/src/main/java/com/braintreepayments/demo/CardFragment.java

```diff
@@ -330,9 +332,10 @@ private void handlePaymentMethodNonceCreated(PaymentMethodNonce paymentMethodNon
             mThreeDSecureRequested = true;
             mLoading = ProgressDialog.show(activity, getString(R.string.loading), getString(R.string.loading), true, false);
 
-            threeDSecureClient.performVerification(activity, threeDSecureRequest(paymentMethodNonce), (request, lookup, error) -> {
-                if (request != null && lookup != null) {
-                    threeDSecureClient.continuePerformVerification(activity, request, lookup, this::handleThreeDSecureResult);
+            ThreeDSecureRequest threeDSecureRequest = threeDSecureRequest(paymentMethodNonce);
+            threeDSecureClient.performLookup(activity, threeDSecureRequest, (threeDSecureResult, error) -> {
+                if (threeDSecureResult != null) {
+                    threeDSecureClient.initiateChallengeWithLookup(activity, threeDSecureRequest, threeDSecureResult, (paymentMethodNonce1, error1) -> handleThreeDSecureResult(paymentMethodNonce1, error1));
```

Comment:
```suggestion
                    threeDSecureClient.initiateChallengeWithLookup(activity, threeDSecureRequest, threeDSecureResult, this::handleThreeDSecureResult);
```
Any reason not to use that syntax? It's a little more concise.